submit_jobs/src/funcs.ts

```diff
@@ -55,15 +60,18 @@ export function getAlgorithms(ticket?:string) {
   });
 }
 
-export function getDefaultValues(code_path) {
+export async function getDefaultValues(state: IStateDB, code_path) {
```

Comment:
I don't think this function needs to get the state.. it just calls the Jupyter backend to do some things that would be easier in Python.  It never calls the MAAP API or uses the username/token.

If you want to leave it in just to standardize passing state to all the requests, that is fine too

Reply:
thanks -- this is removed now.